pkg/edvutils/edvutils.go

```diff
@@ -63,3 +74,98 @@ func Base58Encoded128BitToUUID(name string) (string, error) {
 
 	return storeNameUUID.String(), nil
 }
+
+// CheckIfURI checks if the given string is a valid URI.
+func CheckIfURI(str string) error {
+	_, err := url.ParseRequestURI(str)
+	if err != nil {
+		return fmt.Errorf(messages.InvalidURI, str)
+	}
+
+	return nil
+}
+
+// CheckIfArrayIsURI checks if every string in the given string array is a valid URI.
+func CheckIfArrayIsURI(arr []string) error {
+	for _, str := range arr {
+		if err := CheckIfURI(str); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// ValidateRawJWE returns an error if the given raw JWE is empty or has invalid alg fields.
+func ValidateRawJWE(rawJWE json.RawMessage) error {
```

Comment:
It's not "Raw JWE" - it's just JWE. a json.RawMessage is just a []byte that behaves differently when serialized.

You can call this method just "ValidateJWE".

(You might even consider seeing if the method signature can just use a []byte directly. Not sure if this is really a better thing to do, but try it out and see)

Reply:
I changed it to using []byte directly